### PR TITLE
chore: remove personal config

### PR DESCRIPTION
**Pull request by fisker**
I don't see why we need those package of configs.

If you want introduce new plugins/rules, just add them.

### CODE REVIEW

**Comment by JounQin** [comment]
Because it is easier to maintain access projects, we do have personal configs previously, but those seems unmaintained and very outdated. But I'm fine to be verbose here.

File: .eslintrc.json [hunk 1]
```diff
@@ -4,32 +4,15 @@
     "mocha": true
   },
   "extends": [
-    "@1stg",
-    "plugin:eslint-plugin/recommended"
```
Comment (JounQin) [inline .eslintrc.json:8]: A lot of plugins are missing in this PR.

**Review by JounQin — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (JounQin) [reply .eslintrc.json:8]: And also there some special logic inside

1. https://github.com/1stG/configs/blob/master/packages/config/index.js
2. https://github.com/1stG/configs/blob/master/packages/lint-staged/base.js#L7-L8

File: .prettierrc [hunk 2]
```diff
@@ -1 +1,5 @@
-"@1stg/prettier-config/semi"
+{
```
Comment (JounQin) [inline .prettierrc:1]: https://github.com/1stG/configs/blob/master/packages/prettier-config/base.js#L22-L48

**Review by JounQin — CHANGES_REQUESTED** [review]

**Comment by JounQin** [comment]
Close because a lot of features are missing.

File: .eslintrc.json (hunk 1 above)
Comment (fisker) [reply .eslintrc.json:8]: It's hundreds line project! What for ?

File: .prettierrc (hunk 2 above)
Comment (fisker) [reply .prettierrc:1]: Same as above.

File: .eslintrc.json (hunk 1 above)
Comment (JounQin) [reply .eslintrc.json:8]: That's why I'm using presets here.

**Pull request closed without merging**